scout/server/blueprints/variant/controllers.py

```diff
@@ -326,10 +326,16 @@ def observations(store, loqusdb, case_obj, variant_obj):
 
     institute_id = variant_obj["institute"]
     institute_obj = store.institute(institute_id)
-    obs_data = loqusdb.get_variant(variant_query, loqusdb_id=institute_obj.get("loqusdb_id")) or {}
+    obs_data = (
+        loqusdb.get_variant(variant_query, loqusdb_id=institute_obj.get("loqusdb_id", "default"))
+        or {}
+    )
     if not obs_data:
         LOG.debug("Could not find any observations for %s", composite_id)
-        obs_data["total"] = loqusdb.case_count()
+        obs_data["total"] = loqusdb.case_count(
+            variant_category=variant_obj["category"],
```

Comment:
Do you handle "cancer" and "cancer_SV", especially for the total counts? If so I missed it?

Reply:
Very useful observation. I don't handle it. Will fix!

Reply:
But now I have a doubt: current master shows observations for this cancer variant: https://scout.scilifelab.se/cust087/F0039489-2/04f0a08ba38bf0585c97091997f74223

How comes? "cancer" and "cancer_sv" are passed to the query as they are.. Bah I'll check with a cancer case with observations on stage